File: JAVA_SIS104/app/src/main/java/com/example/prueba_java/GraficoActivity.java
package com.example.prueba_java;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;

import com.example.prueba_java.backend.grafico;

public class GraficoActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //setContentView(R.layout.activity_grafico);
        setContentView(new grafico(GraficoActivity.this));

    }


}

File: KOTLIN_SIS104/app/src/main/java/com/example/prueba_kotlin/GraficoActivity.kt
package com.example.prueba_kotlin

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import com.example.prueba_kotlin.backend.graficos

class GraficoActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        //setContentView(R.layout.activity_grafico)
        setContentView(graficos(this))

        /*val buttonSalirGrafico = findViewById<Button>(R.id.buttonSalirGraf)
        buttonSalirGrafico.setOnClickListener {
            finish()
        }*/
    }
}